blog/2022-02-18-paper-elements.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Zack Barett
+authorURL: https://github.com/zsarnett
+title: "Paper Elements are being removed"
+---
+
+In 2022.3, we are removing all of our `paper-dropdown-menu` and `paper-input` elements in the Frontend Repository. This means that any custom card that was previously using these elements in your cards or card editors will no longer render correctly and will require updates to your codebase.
+
+## Why did we do this?
+
+We are converting our code base from the deprecated `paper` elements to the new [Material Web Components](https://github.com/material-components/material-components-web-components) from Google. This is to keep our frontend up to date with the latest components and features.
+
+## What should custom cards do?
+ 
+You can replace your paper elements with mwc elements but there is no gaurentee that these elements will be loaded when your users load your editor or cards. As well as no guarentee that we use these elements forever.
```

Comment:
We should not suggest this. Instead we should suggest creators to bundle their own components with unique names. This is supported by MWC. See ESP Web Tools as an example: https://github.com/esphome/esp-web-tools/blob/main/src/components/ewt-checkbox.ts

Reply:
```suggestion
Custom Card Creators can bundle their own components with their own unique names. This is now supported by MWC. See ESP Web Tools that use this: https://github.com/esphome/esp-web-tools/blob/main/src/components/ewt-checkbox.ts
```